ci/repo_cleanups/remove_stale_rpm.sh

```diff
@@ -0,0 +1,231 @@
+#!/bin/bash 
+
+
+function run_git {
+#   git for-each-ref --sort=-committerdate refs/remotes --format='%(HEAD)%(refname:short), %(committerdate:relative)'
+    git for-each-ref --sort=-committerdate refs/remotes --format='%(refname:short), %(committerdate:relative)'
+}
+
+
+function write_script {
+
+    var=$(cat <<"EOF"
+
+    declare -A  git_branches    ## Store the list of git branches in a hash
+    stale_weeks=3               ## Any git branches which have not been pushed
+                                ## to within $stale_weeks weeks will be considered
+                                ## "stale".
+
+    rpm_repo_dir=/var/www/html/openvdc-repos
+    
+    function remove_dir {
+        dead_dir=$1
+    
+        here=$PWD
+    
+        if [[ ! "${here}" =  "${rpm_repo_dir}" ]]; then
+            echo "!! Not in the proper directory! Should be in \"${rpm_repo_dir}\"..."
+            exit 1
+        fi
+        
+        for no_rm in "master" "develop"; do
+           if [[ "${dead_dir}" = "${no_rm}" ]]; then
+               echo "Cannot remove \"${no_rm}\". Ignoring."
+               return 0
+           fi
+        done
```

Comment:
Still haven't understood the script completely but relying on what the current working directory is seems a bit of a red flag to me. I would prefer constructing the absolute path to be deleted. We already have the `rpm_repo_dir` variable so it shouldn't be too hard to do.

I plan to read some more to see what exactly is being done and then I'll give more concrete suggestions. :)